astropy/utils/tests/test_introspection.py

```diff
@@ -92,3 +92,26 @@ def a_staticmethod(): pass
     assert not isinstancemethod(MyClass, MyClass.another_classmethod)
     assert not isinstancemethod(MyClass, MyClass.a_staticmethod)
     assert isinstancemethod(MyClass, MyClass.an_instancemethod)
+
+
+def _minversion_test():
```

Comment:
To avoid code repetition when both `pkg_resources.parse_version` and `distutils.version.LooseVersion` are tested.